firebase-vscode/src/workflow.ts

```diff
@@ -129,17 +127,18 @@ export async function setupWorkflow(
    * Call pluginLogger with log arguments received from webview.
    */
   broker.on("writeLog", async ({ level, args }) => {
-    pluginLogger[level]('(Webview)', ...args);
+    pluginLogger[level]("(Webview)", ...args);
```

Comment:
According to the style guide, single quotes are preferred:
go/tsjs-style#use-single-quotes